src/gui/accountstate.cpp

```diff
@@ -171,6 +171,22 @@ AccountState::AccountState(AccountPtr account)
         ownCloudGui::raise();
         msgBox->open();
     });
+
+
+    connect(_account->accessManager(), &AccessManager::isBehindCaptivePortalChanged, this, [this](bool onoff) {
+        if (onoff) {
+            _queueGuard.block();
+        } else {
+            // TODO: empty queue?
```

Comment:
I'm not sure. It might depend on how long we were blocked, 10s or 20h...

Reply:
I'm still not sure what to do with this

Reply:
Lets clear it.